src/Common/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
It was done with a modified version of the tool. I needed to modify the tool because the test .builds files do not map 1-to-1 with what we want in these Configurations.props files.